### PR TITLE
Deposit reagents

### PR DESCRIPTION
**Pull request by Dabbel**
In bank with reagent tab selected, right click deposit normal reagents rather than warbank items

### CODE REVIEW

File: themes/themes.lua [hunk 1]
```diff
@@ -408,7 +408,11 @@ function themes.SetupBagButton(bag, decoration)
       GameTooltip:AddDoubleLine(L:G("Left Click"), L:G("Open Menu"), 1, 0.81, 0, 1, 1, 1)
       GameTooltip:AddDoubleLine(L:G("Shift Left Click"), L:G("Search Bags"), 1, 0.81, 0, 1, 1, 1)
       if addon.isRetail then
-        GameTooltip:AddDoubleLine(L:G("Right Click"), L:G("Deposit Warbank Items"), 1, 0.81, 0, 1, 1, 1)
+        if BankFrame.selectedTab == 2 then
```
Comment (Cidan) [inline themes/themes.lua:411]: Please use `bag.bankTab == const.BANK_TAB.REAGENT` here

File: themes/themes.lua [hunk 2]
```diff
@@ -448,7 +452,11 @@ function themes.SetupBagButton(bag, decoration)
       end
     elseif e == "RightButton" then
       if bag.kind == const.BAG_KIND.BANK and addon.isRetail then
-        C_Bank.AutoDepositItemsIntoBank(Enum.BankType.Account)
+        if BankFrame.selectedTab == 2 then
```
Comment (Cidan) [inline themes/themes.lua:455]: See above comment.

**Review by Cidan — CHANGES_REQUESTED** [review]

**Review by Cidan — APPROVED** [review]
LGTM

**Comment by burntorangejedi** [comment]
I guess what I'm saying is that I'd like to be able to set preferences per category. 

Example 1 - I'm Mining/Herbalism. When I click the "deposit to warbank" button, everything should go to Warbank (current functionality)

Example 2 - I'm Herbalism/Alchemy. When I click the "deposit to warbank" button, everything goes to warbank EXCEPT all my herbs. Those go, instead, to my Reagent bank so I can access them from the crafting tables and the auction house.

Make sense? Is that possible to do today in some form?

**Comment by AMaribo** [comment]
> I guess what I'm saying is that I'd like to be able to set preferences per category.
> 
> Example 1 - I'm Mining/Herbalism. When I click the "deposit to warbank" button, everything should go to Warbank (current functionality)
> 
> Example 2 - I'm Herbalism/Alchemy. When I click the "deposit to warbank" button, everything goes to warbank EXCEPT all my herbs. Those go, instead, to my Reagent bank so I can access them from the crafting tables and the auction house.
> 
> Make sense? Is that possible to do today in some form?

Deposit functionality is tied to whatever Blizzard decides should be deposited when clicking the button.

**Pull request merged**